content/updates/2020-10/4-Data-Quality-Insights-new-criteria-and-ability-to-choose-your-data-quality-focuses.md

```diff
@@ -0,0 +1,15 @@
+---
+pim_announcement_img: ./img/.png
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+---
+# Data Quality Insights new criteria and ability to choose your quality focuses
```

Comment:
Title will be probably too long, it's 77 char included spaces.
The word 'criteria' makes me think of enrichment and consistency... I'd probably need more details on the features to explain them :)